source/manual/deploying.html.md

```diff
@@ -17,27 +17,43 @@ If you are responding to a security incident, follow the steps to [deploy fixes
 [reg]: https://gds.blog.gov.uk/2012/11/02/regular-releases-reduce-risk
 [sec]: /manual/deploy-fixes-for-a-security-vulnerability.html
 
-## 1. Test on integration
+## Deploying a branch
+
+It can be useful to test your branch in a non-local environment before it is merged to master. Note that you should only do this on Integration, unless you have a good reason to deploy to a different environment.
+
+Go to the `Deploy_App` job in Jenkins and click 'Build with Parameters':
+
+- `TARGET_APPLICATION` - choose the name of the repository you want to deploy
+- `DEPLOY_TASK` - usually 'deploy' is most appropriate
+- `TAG` - put the name of your branch
+- Un-check the `NOTIFY_RELEASE_APP` checkbox, as we don't want to notify Release about our branch
```

Comment:
I think it's good to leave this checked, as otherwise release doesn't accurately reflect what the current state of things is.

Reply:
Ahh fair enough, I was worried this would 'contaminate' Release and risk people accidentally promoting branches through the environments. Will change it now.